IyiKuyoro/portfolio
# View Articles

#### What does this PR do?
This pull request adds the ability to view articles on the Web

#### Description of Task to be completed?
- [x] View list of articles
- [x] View a single article
- [x] See 404 page when the article is not found

#### How should this be manually tested?
- Clone the repo and check out this branch
- Build the app and set the backend URL
- Visit the home page and click articles

#### What are the relevant pivotal tracker stories?
#165596513

#### Any background context you want to add?

#### Screenshots
